intezer_analyze_cli/cli.py

```diff
@@ -156,8 +162,9 @@ def analyze_by_list(path):
 @main_cli.command('index', short_help='index a file or a directory')
 @click.argument('path', type=click.Path(exists=True))
 @click.argument('index_as', type=click.STRING)
-@click.argument('family_name', required=False, type=click.STRING)
-def index(path, index_as, family_name=None):
+@click.argument('family_name', required=False, type=click.STRING, default=None)
+@click.option('--ignore-directory-size', is_flag=True, help='ignore directory size limit')
```

Comment:
i mean both mean size, but at first i thought we are filtering by "size" as in how big is the folder, but apprently we're filtering by file count, maybe it should be mentioned somewhere because "size" is confusing